<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdersMenu</class>
 <widget class="QWidget" name="OrdersMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>833</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>833</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Управление Заказами</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
  </property>
  <widget class="QPushButton" name="AddOrder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>138</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить заказ</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>281</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="NumOfRowLayout">
    <property name="spacing">
     <number>25</number>
    </property>
    <item>
     <widget class="QLabel" name="NumOfRowLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Номер строки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="NumOfRowEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="MainTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>132</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>Заказы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveOrder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>540</y>
     <width>138</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить заказ</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RegNumClientLayout">
    <property name="spacing">
     <number>55</number>
    </property>
    <item>
     <widget class="QLabel" name="RegNumClientLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Рег. Номер клиента</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RegNumClientEdit">
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="OrdersTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>771</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RegNumMasterLayout">
    <property name="spacing">
     <number>55</number>
    </property>
    <item>
     <widget class="QLabel" name="RegNumMasterLabel">
      <property name="maximumSize">
       <size>
        <width>145</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Рег. Номер мастера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RegNumMasterEdit">
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ThingLayout">
    <property name="spacing">
     <number>45</number>
    </property>
    <item>
     <widget class="QLabel" name="ThingLabel">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Техника для ремонта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ThingEdit">
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ExecuteOrder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>540</y>
     <width>138</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выполнить заказ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
